akka-actor-typed/src/main/scala/akka/actor/typed/StatusResponse.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.actor.typed
```

Comment:
could be placed in `akka-actor` for interop reasons, and maybe future deprecation of `Status`

Reply:
good if we deprecate Status in favour of this variant that play better with typed.